examples/metrics/grafana-dashboards/strimzi-operators.json

```diff
@@ -0,0 +1,1705 @@
+        "avg": false,
+        "current": false,
+        "hideEmpty": false,
+        "hideZero": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "links": [],
+      "nullPointMode": "null",
+      "paceLength": 10,
+      "percentage": false,
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "sum(increase(strimzi_reconciliations_locked_total[1h])) by (kind)",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "{{kind}}",
+          "refId": "A"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeRegions": [],
+      "timeShift": null,
+      "title": "Reconciliation without Lock per hour",
+      "tooltip": {
+        "shared": true,
+        "sort": 0,
+        "value_type": "individual"
+      },
+      "type": "graph",
+      "xaxis": {
+        "buckets": null,
+        "mode": "time",
+        "name": null,
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
+        {
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        },
+        {
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "cacheTimeout": null,
+      "dashLength": 10,
+      "dashes": false,
+      "decimals": null,
+      "fill": 1,
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 0,
+        "y": 16
+      },
+      "id": 47,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "hideEmpty": false,
+        "hideZero": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "links": [],
+      "nullPointMode": "null",
+      "paceLength": 10,
+      "percentage": false,
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "sum(increase(strimzi_reconciliations_total[1h])) by (kind)",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "{{kind}}",
+          "refId": "A"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeRegions": [],
+      "timeShift": null,
+      "title": "Started Reconciliation per hour",
```

Comment:
I assume it's the number of reconciliation started due to added, updated or deleted CRs. Maybe just "Started" is not enough to understand, at least for me. Something around "CRs triggered Reconciliation per hour" ... but I have not strong opinion, just that "Started" doesn't clarify the nature.

Reply:
The are not just the reconciliations triggered by the CR - such as added or modified. There are also the periodical once. There are 5 counters in general:
* The periodical reconciliations are fired everytime the timer fires .... increment by 1 for a single periodical reconcilition regardless the number of resources.
* The started reconciliations covers both watch and periodical. But is per resource. So if you have 2 Kafka clusters, they each increment by one
* Successful are the started which ended ok
* Failed are the started whithc failed with error other than no lock
* And the locked are those which didn't got the lock (tracked separately because they are not really errors per se)

Reply:
Yeah, these all the first 5 graphs in the picture but focusing on the "started ...." it could be not clear compared to the periodical one (even because you said that it contains the periodical one but is per resource). Maybe we can exclude the periodical ones in this "started ... " metric ?

Reply:
No, we cannot exclude it easily. It is started because the counter is increased at startup. Where as the successful / faled / locked are at the end. I think you also need to count all the reconciliations to be able to corelate the started against the successful/failed/locked.